unittests/state_history_tests.cpp

```diff
@@ -169,18 +169,18 @@ BOOST_AUTO_TEST_CASE(test_trace_log) {
    BOOST_REQUIRE(!std::holds_alternative<eosio::ship_protocol::prunable_data_type::none>(
        get_prunable_data_from_traces(traces, cfd_trace->id)));
 
-   std::vector<transaction_id_type> ids{cfd_trace->id};
-   log.prune_transactions(cfd_trace->block_num, ids);
-   BOOST_REQUIRE(ids.empty());
+//    std::vector<transaction_id_type> ids{cfd_trace->id};
```

Comment:
Any reason to save this commented out code?

Reply:
> Any reason to save this commented out code?

Pushed new commit for uncommenting these lines.